src/components/pages/Books.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Book from './Book';
+import Form from './Form';
+
+const bookList = [
+  {
+    id: 'book1',
+    title: 'Things Fall Apart',
+    author: 'Chinua Achebe'
+  },
+  {
+    id: 'book2',
+    title: 'Life is Good',
+    author: "Adelakun Abdulwasi'"
+  }
+];
+
+const Books = () => (
+  <>
+    <div className="container">
+      <h1 className="booklist-header">List of Books</h1>
+
+      {bookList.map((book) => (
+        <div className="booklist" key={book.id}>
```

Comment:
you can move this into the <Book /> component for and pass the key prop to the book component. Do not worry about prop-types error, it won't flag the key prop

Reply:
Done